internal/pkg/app/handlers_v1_test.go

```diff
@@ -0,0 +1,24 @@
+package app
+
+import (
+	"encoding/json"
+	goldie "github.com/sebdah/goldie/v2"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestVersionHandler(t *testing.T) {
+	Version = "v1.0.2"
```

Comment:
Modifying global variables in testing can be problematic.  The app itself could take this value as an argument upon initialization.  This eliminates `go test -race` issues.

Reply:
In general, I agree with you. In fact, I think we should avoid global variables. I do think for "Version" is a special case (and really just a read-only value), we will probably ever going to have just this one test case to test out Version making `go test -race` irrelevant. We can refactor the code later if needs to be

Reply:
k